server/config_push_proxy.go

```diff
@@ -12,16 +12,21 @@ import (
 )
 
 type ConfigPushProxy struct {
```

Comment:
The same way we have an example config file, can we add an example config file for the logger?

Reply:
@larkox What do you mean exactly? For the old logger or mlog.Logger?

Reply:
For the new logger. We are asking the user of the push proxy to use a new file, so we need to provide documentation on what that file looks like. A example file (like the one in `config/mattermost-push-proxy.sample.json`, but for the logger) would be great. We also need to add documentation for that, but I guess that can be a link to the mlog documentation, not sure.

Now that I think about it, we should update that file with the new fields (maybe remove the fields we no longer use) to reflect these new values.